code/src/Components/Select.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+import { Progress } from 'Components/Progress'
+
+export const Select = ({ food, setFood }) => {
+    return (
+        <form>
+            <Progress current='3' total='3'/> 
+            <h2>Here we go: select your favourite sweet from the dropdown! Oops! You can choose only one Sweetie!</h2>
+            <select
+                onChange={event => setFood(event.target.value)}
+                value={food}
+                required
+            >
```

Comment:
When we have a boolean attribute, it's convention to start with that:
```
<select
  required
  onChange={event => setFood(event.target.value)}
  value={food}
>
```

Reply:
I see, going to be fixed!